src/dispatch.c

```diff
@@ -371,6 +371,8 @@ void qd_dispatch_free(qd_dispatch_t *qd)
     qd_python_finalize();
     qd_dispatch_set_router_id(qd, NULL);
     qd_dispatch_set_router_area(qd, NULL);
+
+    free(qd);
```

Comment:
this is a funny issue; dispatch does not free this, but because there is a global variable pointing to the current qd_router_t, it is not detected as a leak; when I want to start the router multiple times in a single process, subsequent initializations overwrite the global, and the leak gets reported